parfive/downloader.py

```diff
@@ -563,6 +577,7 @@ async def _get_http(self, session, *, url, filepath_partial, chunksize=None,
             return str(filepath)
 
         except Exception as e:
+            remove_file(get_filepath(filepath_partial(None, url), False)[0])
```

Comment:
By the time we get here, if `filepath` isn't defined we can't have written anything to the disk at that path, so we don't have anything to delete.

Reply:
I have no idea where I should even check for a file and remove, so this was my only shot.

Reply:
I think the location is fine, but you should be able to use the `filepath` variable?